### PR TITLE
Partitioned FSI updates and improvements

### PR DESCRIPTION
**Pull request by rubenzorrilla**

### CODE REVIEW

File: applications/FSIapplication/fsi_application.cpp [hunk 1]
```diff
@@ -28,6 +28,7 @@ KRATOS_CREATE_VARIABLE(double, MAPPER_SCALAR_PROJECTION_RHS);
 KRATOS_CREATE_VARIABLE(double, SCALAR_PROJECTED);
 KRATOS_CREATE_VARIABLE(double, FICTITIOUS_FLUID_DENSITY);
 KRATOS_CREATE_VARIABLE(double, FSI_INTERFACE_RESIDUAL_NORM);
+KRATOS_CREATE_VARIABLE(double, FSI_INTERFACE_MESH_RESIDUAL_NORM);
```
Comment (roigcarlo) [inline applications/FSIapplication/fsi_application.cpp:31]: Is created, defined, and registered in python but not in kratos.

Maybe missing:
```C++
KRATOS_REGISTER_VARIABLE(FSI_INTERFACE_MESH_RESIDUAL_NORM);
```

Comment (rubenzorrilla) [reply applications/FSIapplication/fsi_application.cpp:31]: Done! Thanks Charlie!

File: applications/FSIapplication/custom_utilities/partitioned_fsi_utilities.hpp [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+                }
+            }
+
+            // Accumulate the obtained consistent residual values
+            VectorType ConsResVect(BlockSize*NumNodes);
+            TSpace::Mult(MassMat, ResVect, ConsResVect);
+
+            for (int ii=0; ii<static_cast<int>(NumNodes); ++ii)
+            {
+                array_1d<double, 3> aux_val;
```
Comment (RiccardoRossi) [inline applications/FSIapplication/custom_utilities/partitioned_fsi_utilities.hpp]: array_1d<double, 3> aux_val = ZeroVector(3);

**Review by RiccardoRossi — APPROVED** [review]
looks fine to me

i just made a minor stylistic comment

File: applications/FSIapplication/custom_utilities/partitioned_fsi_utilities.hpp (hunk 2 above)
Comment (jcotela) [reply applications/FSIapplication/custom_utilities/partitioned_fsi_utilities.hpp]: array_1d<double, 3> aux_val(3,0.0);
Edit: sorry, I commented before seeing this had already been merged. I still think avoiding the ublas thing is better though.

**Pull request merged**